pkg/daemons/executor/etcd.go

```diff
@@ -3,8 +3,13 @@
 package executor
 
 import (
+	"github.com/rancher/k3s/pkg/version"
```

Comment:
Can you run goreturns, go imports, or gofmt on this to get the imports formatted to standards?

Reply:
done